src/components/popup-dialog/PopupDialog.js

```diff
@@ -38,8 +38,9 @@ const PopupDialog = ({ content, closeModal }) => {
 
   return (
     <Dialog
-      fullScreen={ (width < 600) }
-      maxWidth="xl" onClose={ onClose }
+      data-testid='popup'
+      fullScreen={ (width < 600) } maxWidth="xl"
```

Comment:
You can add useBreakpoints and do this: fullScreen={ (size === 'mobile') }

Reply:
done